src/Resources/config/tool.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <parameters>
+        <parameter key="stof_doctrine_extensions.tool.actor_provider.class">Stof\DoctrineExtensionsBundle\Tool\TokenStorageActorProvider</parameter>
```

Comment:
Please put the class directly on the service instead. Defining service classes through parameters was common in the early days of Symfony, but it is not considered a best practice anymore as it is generally a very bad extension point (and it bloats the list of parameters)

Reply:
Updated (most of this was just copy paste from what else was there so I didn't want to get too opinionated about using best practices 😅)